src/app/hearing-part/components/listing-create/listing-create.component.ts

```diff
@@ -218,7 +218,8 @@ export class ListingCreateComponent implements OnInit {
                 reservedJudgeId: undefined,
                 communicationFacilitator: undefined,
                 userTransactionId: undefined,
-                numberOfSessions: 1
+                numberOfSessions: 1,
+                isMultiSession: false,
```

Comment:
should we also update the isMultiSessionListing() method?

Reply:
Done